core/src/tile/tileManager.cpp

```diff
@@ -15,16 +15,16 @@
 
 namespace Tangram {
 
-TileManager::TileManager(TileTaskQueue& _tileWorker) :
+TileManager::TileManager(std::shared_ptr<Platform> platform, TileTaskQueue& _tileWorker) :
     m_workers(_tileWorker) {
 
     m_tileCache = std::unique_ptr<TileCache>(new TileCache(DEFAULT_CACHE_SIZE));
 
     // Callback to pass task from Download-Thread to Worker-Queue
-    m_dataCallback = TileTaskCb{[this](std::shared_ptr<TileTask> task) {
+    m_dataCallback = TileTaskCb{[this, &platform](std::shared_ptr<TileTask> task) {
```

Comment:
Difference concerning access time to the pointer? I would say that it's fairly similar, increase usage count is done only once if passed by copy, no increment should be done on usage count if passed by ref.

It's good you pointed that out, it's a leftover from a previous implementation where a reference was used, and it shouldn't be safe to pass it by reference here considering the scope of `platform`. Will update.